yarn.lock

```diff
@@ -1171,6 +1171,13 @@
   dependencies:
     regenerator-runtime "^0.13.11"
 
+"@babel/runtime@^7.17.2":
```

Comment:
Hm, we should avoid installing `babel/runtime` twice. Are we sure we need a downgraded version of what is already present?

Reply:
It's a dependency of our libraries and is managed by `yarn`